<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-07-15T14:18:03Z" name="Default suite" finished-at="2019-07-15T14:19:19Z" duration-ms="76484">
    <groups>
    </groups>
    <test started-at="2019-07-15T14:18:03Z" name="Default test" finished-at="2019-07-15T14:19:19Z" duration-ms="76484">
      <class name="testcases.Sigin_Test">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:18:03Z" name="beforeSuite" finished-at="2019-07-15T14:18:03Z" duration-ms="151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest(org.testng.ITestContext)[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:18:03Z" name="beforeTest" finished-at="2019-07-15T14:18:16Z" duration-ms="13103" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@29be7749]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:18:16Z" name="setUp" finished-at="2019-07-15T14:18:16Z" depends-on-methods="wdMethods.ProjectMethods.beforeTest" duration-ms="29" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:18:16Z" name="beforeClass" finished-at="2019-07-15T14:18:16Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="signInToTheApplication()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:18:16Z" name="signInToTheApplication" finished-at="2019-07-15T14:19:17Z" duration-ms="60890" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[FAILED]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: FAILED
at utils.Reporter.reportStep(Reporter.java:46)
at utils.Reporter.reportStep(Reporter.java:55)
at wdMethods.ApMethods.click(ApMethods.java:252)
at pages.DashboardPage.accessMyAccountPage(DashboardPage.java:90)
at testcases.Sigin_Test.signInToTheApplication(Sigin_Test.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInToTheApplication -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:18:16Z" name="beforeMethod" finished-at="2019-07-15T14:18:16Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:19:17Z" name="afterMethod" finished-at="2019-07-15T14:19:18Z" duration-ms="617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:19:18Z" name="beforeMethod" finished-at="2019-07-15T14:19:18Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="signInToTheApplication2()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:19:18Z" name="signInToTheApplication2" finished-at="2019-07-15T14:19:19Z" duration-ms="1273" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[FAILED]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: FAILED
at utils.Reporter.reportStep(Reporter.java:46)
at utils.Reporter.reportStep(Reporter.java:55)
at wdMethods.ApMethods.type(ApMethods.java:227)
at pages.SignInPage.signIn(SignInPage.java:57)
at testcases.Sigin_Test.signInToTheApplication2(Sigin_Test.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInToTheApplication2 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:19:19Z" name="afterMethod" finished-at="2019-07-15T14:19:19Z" duration-ms="552" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:19:19Z" name="afterTest" finished-at="2019-07-15T14:19:19Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:testcases.Sigin_Test@1184ab05]" started-at="2019-07-15T14:19:20Z" name="afterSuite" finished-at="2019-07-15T14:19:20Z" duration-ms="217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.Sigin_Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
